pkg/volume/hostpath/host_path.go

```diff
@@ -366,7 +366,11 @@ func (ftc *fileTypeChecker) IsFile() bool {
 	if !ftc.Exists() {
 		return false
 	}
-	return !ftc.IsDir()
+	pathType, err := ftc.mounter.GetFileType(ftc.path)
```

Comment:
did we at least get the fix in on it's own? @dims @dixudx

Reply:
poke @dims @dixudx @msau42

Reply:
you mean in k8s.io/utils?

Reply:
The PR was very close to merging last time, the last thing remaining was to double check that the tests are creating pods that are getting decently spread out across nodes to not overload a single node.

We've also fixed a number of general test issues that were causing flakes and timeouts in the past.

@dixudx do you have time to rebase?